### ISSUE TITLE
javascript_template.js.erb not being interpreted as JavaScript

### ISSUE THREAD

**Issue by maximsko** [pitr/angular-rails-templates#124]
`/lib/angular-rails-templates/javascript_template.js.erb` is not being interpreted as JS I think, because I have the plain output of the file in my webbrowser:

```javascript
// Angular Rails Template // source: app/assets/javascripts/angular/sessions/_component_single.html.slim angular.module("templates").run(["$templateCache", function($templateCache) { $templateCache.put("angular/sessions/_component_single.html", '
```

then actually comes the template which is fine, and at the end I see the ending brackets again:

```
') }]);
```

**Comment by brilliantdeveloping** [comment]
same issue

**Comment by pitr** [comment]
What version of rails and angular-rails-templates are you using?

**Comment by maximsko** [comment]
I think it was the absolute latest by the time of writing for both. I am going to try it now again to see if the issue is still present.

**Comment by maximsko** [comment]
Seems to be fixed for `angular-rails-templates 1.0.0` with `rails 4.2.6`

**Comment by pitr** [comment]
sounds good
